notebooks/nessie-iceberg-flink-demo-nba.ipynb

```diff
@@ -407,7 +407,7 @@
         {
           "name": "stdout",
           "output_type": "stream",
-          "text": "\u001b[33m* main  2e1cfa82b035c26cbbbdae632cea070514eb8b773f616aaeaf668e2f0be8f10d comment\n\u001b[0m  dev   3f743d120c8b4898237b19f7e82d3193bea7c5cfb6e1ffb4b8671de548f70422 comment\n\n"
+          "text": "  dev   c2bf8f664c7d669f8c17cb2ebf5dbfe00a764bd6f8b0d15eb764dddf0b8aff81\n\u001b[33m* main  2e1cfa82b035c26cbbbdae632cea070514eb8b773f616aaeaf668e2f0be8f10d\n\u001b[0m\n"
```

Comment:
why would it be a bad idea to sort the output lines of `nessie --verbose branch` by branch name? (keeping default branch first).

similar for `nessie content list`, right now we get:
```
ICEBERG_TABLE:\n\tnba.totals_stats\n\tnba.salaries\n\n
```
why cant we sort these tables by name?
this would be for display/output purposes only of course.

Reply:
@dimas-b didn't you add some output ordering a while ago to the python client for some stuff? I forgot in which context this was exactly

Reply:
@XN137 too many entries, potentially